cypress/integration/models/developer/applicationinventory/analysis.ts

```diff
@@ -157,7 +176,7 @@ export class Analysis extends Application {
             cy.contains("button", "Next", { timeout: 200 }).click();
             this.selectTarget(this.target);
             cy.contains("button", "Next", { timeout: 200 }).click();
-            if (!this.scope) cy.contains("button", "Next", { timeout: 200 }).click();
+            this.scopeSelect();
```

Comment:
@sshveta If we doesn't gave any scope options (as implied with IF statements) and simply wanted to click on Next button. The Test cases won't fail. As in scopeSelect method I've `cy.contains("button", "Next", { timeout: 200 }).click();` as a compulsory statement. Added in scopeSelect because everything on scope page wanted to be in same method.
For ex. I ran an source analysis just with the default option and Next, the Tests ran Successfully.
![Screenshot from 2022-11-22 14-17-30](https://user-images.githubusercontent.com/87791967/203711621-286d45c9-527c-410e-be3e-5aeaf4a9111f.png)

Reply:
But still if you want will keep as ```
if (this.scope) this.scopeSelect();
cy.contains("button", "Next", { timeout: 200 }).click();
```

Reply:
Ok I see that next click is outside the if scope loop .